docs/services.md

```diff
@@ -1448,7 +1448,7 @@ See the [public documentation][230].
   - `config.language` **[Array][206]&lt;[string][197]>?** Specify the language to use for response text and, for forward geocoding, query result weighting.
      Options are [IETF language tags][233] comprised of a mandatory
      [ISO 639-1 language code][234] and optionally one or more IETF subtags for country or script.
-  - `config.routing` **[boolean][198]?** Specify whether to request additional metadata about the recommended navigation destination. Only applicable for address features.
+  - `config.routing` **[boolean][198]** Specify whether to request additional metadata about the recommended navigation destination. Only applicable for address features. (optional, default `false`)
```

Comment:
I didn't make this change; I assume it came in through the pre-commit hooks. It seems to be related to the change that was made in the latest commit on master. Happy to just leave it in here or have you handle it separately and I can rebase my fork against that change; just let me know.

Reply:
That's fine, the docs are generated automatically and for various reasons (like the person before not running it) there can be unexpected changes. I wouldn't worry about it.